CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 See full changelog for the OpenAPI schema (OAS) [here](https://github.com/plaid/plaid-openapi/blob/master/CHANGELOG.md).
 
+# 17.0.0
```

Comment:
this looks like the same version from last time 🤔 - should this be 17.1.0?

Reply:
Oops, I think you're right